crab/crab_script_ggOnly.py

```diff
@@ -23,7 +23,7 @@
 analysis=sys.argv[3].split("=")[1]
 
 
-selection='''Sum$(Photon_pt > 18 && (Photon_isScEtaEB || Photon_isScEtaEE) && Photon_electronVeto > 0.5 && Photon_hoe < 0.08) > 1'''
+selection='''Sum$(Photon_pt > 18 && (Photon_isScEtaEB || Photon_isScEtaEE) && Photon_electronVeto < 0.5 && Photon_hoe < 0.08) > 1'''
```

Comment:
With this line, we are changing the configuration, instead of adding a new one. If adding a new configuration in a configurable way (e.g. a command line argument) is difficult/tedious, I believe that the most clear way to add the new configuration would be to keep the previous line (the one you changed) and add below it, commented it out, the new line as above, with a clear comment what its usefulness is.